src/test/java/reactor/netty/udp/UdpServerTests.java

```diff
@@ -221,8 +224,8 @@ private boolean isMulticastEnabledIPv4Interface(NetworkInterface iface) {
 			return false;
 		}
 
-		for (Enumeration<InetAddress> i = iface.getInetAddresses();
-		     i.hasMoreElements(); ) {
+		// Suppressed "JdkObsolete" is deliberate
```

Comment:
just to clarify, maybe mention that the use of _Enumeration_ is deliberate

Reply:
fixed